src/components/Calls/CallIFrame.js

```diff
@@ -7,8 +7,28 @@ import styles from './styles.scss';
 
 
 export const CallIframe = () => {
-	const { token, room } = store.state;
-	const url = `${ Livechat.client.host }/meet/${ room._id }?token=${ token }`;
+	const { token, room, incomingCallAlert } = store.state;
+	const url = `${ Livechat.client.host }/meet/${ room._id }?token=${ token }&layout=embedded`;
+	window.handleIframeClose = async () => store.setState({ incomingCallAlert: {
+		show: false,
+		callProvider: incomingCallAlert.callProvider,
+		callerUsername: incomingCallAlert.callerUsername,
+		rid: incomingCallAlert.rid,
+		time: incomingCallAlert.time,
+	} });
```

Comment:
Oopps... I totally forgot about this earlier :see_no_evil: -  We're maintaining another variable called `ongoingCall` here as well, so we perhaps need to update that too, when the call is either closed, or expanded to new window

Reply:
Update what value in ongoingCall?

Reply:
@Deepak-learner Can provide much better answer to this, however let me try
1. When call stopped - we reset this property so it would be `undefined`
2. When call is opened in new tab - we add some new status there - `ongoingCallInNewTab` (Checkout Deepaks this PR for more info - https://github.com/RocketChat/Rocket.Chat.Livechat/pull/629/files)

Reply:
@Deepak-learner could you please confirm both the points? I'll only proceed after your confirmation.

Reply:
For the first point :There is no need to set property ongoingCallStatus undefined here. As even if the call stopped, we are setting callStatus value with ended. 

For the second point: Yes I agree with @murtaza98 . We can update callStatus value with ongoingCallInNewTab.